### DISCUSSION TITLE
go2rtc frame.jpeg url shows a 404 error

### DISCUSSION THREAD

**Discussion by vajonam** [blakeblackshear/frigate#6094] [Ask A Question] [answered]
I would like to use the frame.jpeg URL from go2rtc, from what I understand it should just work.

this is the link it generated from go2rtc dashboard `https://frigate.domain.com/live/webrtc/api/frame.jpeg?src=trident` however the frame in mp4 format works, am i missing any configs. I cant seem to find anything on how to specifically enable this ?

**Comment by NickM-27** [comment, marked as answer]
That endpoint is only supported with an mjpeg source

**Reply by vajonam** [reply]
I see, is there a clever way of getting a snapshot from a go2rtc only stream? I need to get one for a notification when a door opens or closes. I send out a notification with a picture just to confirm the door opened or closed.

**Reply by vajonam** [reply]
I can use url direcly into the camera, maybe that is best way to go.

**Reply by NickM-27** [reply]
why not just use frigate's latest.jpg api?

**Reply by vajonam** [reply]
this is not frigate camera, just a go2trc stream, defined only for viewing, the stream is not being decoded. go2rtc is just copying the stream for webrtc for display in HA.

**Comment by vajonam** [comment]
```
    garage:
      - rtsp://ip_camera00:554/live2.sdp
```
